dotnet/src/InternalUtilities/src/Schema/JsonSchemaMapper.STJv8.cs

```diff
@@ -243,7 +243,7 @@ private static JsonSchema MapJsonSchemaCore(
                 List<string>? required = null;
                 JsonSchema? additionalProperties = null;
 
-                if (typeInfo.UnmappedMemberHandling is JsonUnmappedMemberHandling.Disallow)
+                if (typeInfo.UnmappedMemberHandling is JsonUnmappedMemberHandling.Disallow || state.Configuration.AdditionalProperties is false)
```

Comment:
I don't see this check on the original reference. (As I understand we shouldn't change this source unless applies for the library as a whole).

@eiriktsarpalis, Let us know if this change can be made (and make sense to the logic for the STJv8. 


https://github.com/eiriktsarpalis/stj-schema-mapper/blob/58360933979019c1160794cf909aba296dc710da/src/JsonSchemaMapper/JsonSchemaMapper.STJv8.cs#L248

Reply:
Ahh, we should specify at these files headers that it's copied from some source, so we shouldn't modify it. @eiriktsarpalis thanks for the example, I will try to use it and let you know if that works.

Reply:
`JsonSchemaMapperConfiguration.TransformSchemaNode` works well, @eiriktsarpalis thanks for the suggestion!